### ISSUE TITLE
ControlBridge causing needless regeneration

### ISSUE THREAD

**Issue by enn-nafnlaus** [ltdrdata/ComfyUI-Impact-Pack#653]
Simple demonstration attached.

[ControlBridgeProblem.json](https://github.com/user-attachments/files/15981194/ControlBridgeProblem.json)

It's easy enough to recreate.

1) Load an upscale model using "Load Upscale Model"
2) Route the upscale model to a control bridge
3) Route the output of the control bridge to an "Upscale Image (using Model)".
4) Give the upscaler an empty or loaded image to upscale and a preview image output.

Now keep clicking Queue Prompt.  It will generate an image every time.

Now eliminate the Control Bridge from the workflow - route the model straight to the upscaler.  It will generate once the first time, but after that, it will use the cached image and not force regeneration (the proper behavior).

In short, the ControlBridge is breaking this caching behavior, which can lead to greatly slowing down complex workflows - something Control Bridge is, ironically, designed to prevent  :)

**Comment by ltdrdata** [comment]
This is patched.

**Comment by enn-nafnlaus** [comment]
> This is patched.

Great, thanks - will try it out this evening!  :)

**Comment by zslefour** [comment]
<img width="869" height="319" alt="Image" src="https://github.com/user-attachments/assets/6f88969e-5263-4d0d-a5db-d640580dc4aa" />

The problem still exists; when the control variable changes from true to false, it causes three respawns.
